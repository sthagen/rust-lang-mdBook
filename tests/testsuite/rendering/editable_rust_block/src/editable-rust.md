# Chapter 1

```rust,editable
fn f() {
    println!("hello");
}
```

```rust
// Not editable.
```
