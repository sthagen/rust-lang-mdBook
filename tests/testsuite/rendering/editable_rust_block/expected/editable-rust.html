<h1 id="chapter-1"><a class="header" href="#chapter-1">Chapter 1</a></h1>
<pre class="playground"><code class="language-rust editable">fn f() {
    println!("hello");
}</code></pre>
<pre class="playground"><code class="language-rust"><span class="boring">#![allow(unused)]
</span><span class="boring">fn main() {
</span>// Not editable.
<span class="boring">}</span></code></pre>